### PR TITLE
Created Github actions workflow for testing successful integration

### PR DESCRIPTION
**Pull request by ehaligow**
Created Github actions workflow for testing successful integration between ODIM's services and Device Manager.

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+    - name: Generate certs
+      run: |
+        sed -i 's/URP/device-manager/' /tmp/certs/generate_odimra_cert.sh
+        cp /tmp/certs/* build/certs
+        cd $GITHUB_WORKSPACE/build/certs
+        chmod +x generate_odimra_cert.sh 
+        ./generate_odimra_cert.sh deviceManager
+        
+    - name: Install Device Manager and ODIM's services
```
Comment (MaciejMis) [inline .github/workflows/integration.yml:41]: There is a missing Device Manager config update step - password are being removed from source code, so you will have to add them here, as an additional step.

Comment (ehaligow) [reply .github/workflows/integration.yml:50]: done.

**Comment by MaciejMis** [comment]
+1

**Comment by MaciejMis** [comment]
Accepted on 8.08.2023 community meeting

**Pull request merged**